### PR TITLE
[chore] Rename pipelines and centralise the security ones

### PR DESCRIPTION
**Pull request by kang-makes**
As the first iteration to standardize the pipelines, I need all the pipelines to be named the same way

I'll need to run scripts to analyze them such as `diff` and `yq` to see the differences between them and having them with different names makes the scripts a mess.

This is the bare minimum start even if the effort to standardize the pipelines finally does not happen.

This PR also renames the reusable pipelines with the prefix `reusable_` so we can have more readable pipelines as this repository is expected to grow a lot in the near future.

### CODE REVIEW

File: renovate-base.json5 [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
-    "config:base"
+    "config:recommended"
```
Comment (marcsanmi) [inline renovate-base.json5:4]: Why this?

**Review by marcsanmi — APPROVED** [review]
LGTM 👍

File: .github/workflows/reusable_trigger_prerelease.yaml
Comment (sigilioso) [inline .github/workflows/reusable_trigger_prerelease.yaml:1]: Renaming this Ok as long as we don't move the `v1` tag, right? Usage example: https://github.com/newrelic/nri-postgresql/blob/master/.github/workflows/trigger_prerelease.yml#L14

Comment (kang-makes) [reply .github/workflows/reusable_trigger_prerelease.yaml:1]: The PRs that I have opened rename that pipeline too: https://github.com/newrelic/nri-postgresql/pull/140/files#diff-b47f0a5e492987499371b7c8b4e1b28b6b743f77247f7bb6beb7aa767b84b8f3L14-R14

File: renovate-base.json5 (hunk 1 above)
Comment (kang-makes) [reply renovate-base.json5:4]: This is a new pipeline that I have added and that I have added here: https://github.com/newrelic/coreint-automation/pull/33/files#diff-efda70da6ee709cf755707663c052d5f163fa082bb1e6c441fb08cd36d3341f5R24

Renovate bot changed that and the bot complained that "base" does not exist anymore and that it is being migrated to "recommended".

I changed it to have a cleaner CI run.

File: .github/workflows/reusable_trigger_prerelease.yaml
Comment (kang-makes) [reply .github/workflows/reusable_trigger_prerelease.yaml:1]: Another option is to call it a braking change and change it to `v2`. As you wish.

Comment (sigilioso) [reply .github/workflows/reusable_trigger_prerelease.yaml:1]: I'd use `v2` in order to make PRs independent. No strong opinion, though

**Review by sigilioso — APPROVED** [review]
👍

File: .github/workflows/reusable_trigger_prerelease.yaml
Comment (kang-makes) [reply .github/workflows/reusable_trigger_prerelease.yaml:1]: Done :)

**Pull request merged**